clients/src/main/java/org/apache/kafka/clients/admin/internals/DeleteConsumerGroupOffsetsHandler.java

```diff
@@ -104,15 +104,18 @@ public String apiName() {
 
         final Errors error = Errors.forCode(response.data().errorCode());
         if (error != Errors.NONE) {
-            handleError(groupId, error, failed, unmapped);
+            handleGroupLevelError(groupId, error, failed, unmapped);
         } else {
             final Map<TopicPartition, Errors> partitions = new HashMap<>();
             response.data().topics().forEach(topic -> 
-                topic.partitions().forEach(partition -> {
-                    Errors partitionError = Errors.forCode(partition.errorCode());
-                    if (!handleError(groupId, partitionError, failed, unmapped)) {
-                        partitions.put(new TopicPartition(topic.name(), partition.partitionIndex()), partitionError);
+                topic.partitions().forEach(partitionOffsetDeleteResponse -> {
+                    Errors partitionError = Errors.forCode(partitionOffsetDeleteResponse.errorCode());
+                    TopicPartition tp = new TopicPartition(topic.name(), partitionOffsetDeleteResponse.partitionIndex());
+                    if (log.isDebugEnabled() && partitionError != Errors.NONE) {
+                        log.debug("`{}` request for group {} returned error {} in the partition {}.",
+                            apiName(), groupId, partitionError, tp);
                     }
+                    partitions.put(tp, partitionError);
```

Comment:
For partition level error, we don't handle it, just put them into completion result.